packages/x-components/src/x-modules/tagging/wiring.ts

```diff
@@ -255,5 +265,8 @@ export const taggingWiring = createWiring({
   },
   UserClickedADisplayResult: {
     trackDisplayClickedWire
+  },
+  UserClickedADisplayResultWithQuery: {
+    setQueryTaggingFromQueryPreview
```

Comment:
Aren't these two events basically the same? We can use just `UserClickedADisplayResult` with the `resultLinkMetadataPerEvent` in its metadata and make it also call `setQueryTaggingFromQueryPreview` when it's listened:

```suggestion
  UserClickedADisplayResult: {
    trackDisplayClickedWire,
    setQueryTaggingFromQueryPreview
```